docker/docker-compose.yml

```diff
@@ -260,3 +260,31 @@ services:
       - ${MF_COAP_ADAPTER_PORT}:${MF_COAP_ADAPTER_PORT}/tcp
     networks:
       - mainflux-base-net
+
+  twins:
+    image: mainflux/twins:latest
+    container_name: mainflux-twins
+    depends_on:
+      - authn
+      - nats
+    restart: on-failure
+    environment:
+      MF_TWINS_LOG_LEVEL: ${MF_TWINS_LOG_LEVEL}
+      MF_TWINS_DB_NAME: ${MF_TWINS_DB_NAME}
+      MF_TWINS_DB_PORT: ${MF_TWINS_DB_PORT}
+      MF_TWINS_DB_USER: ${MF_TWINS_DB_USER}
+      MF_TWINS_DB_HOST: ${MF_TWINS_DB_HOST}
+      MF_TWINS_HTTP_PORT: ${MF_TWINS_HTTP_PORT}
+      MF_AUTHN_URL: authn:${MF_AUTHN_GRPC_PORT}
+      MF_JAEGER_URL: ${MF_JAEGER_URL}
+      MF_TWINS_THING_ID: ${MF_TWINS_THING_ID}
+      MF_TWINS_THING_KEY: ${MF_TWINS_THING_KEY}
+      MF_TWINS_CHANNEL_ID: ${MF_TWINS_CHANNEL_ID}
+      MF_NATS_URL: ${MF_NATS_URL}
+      MF_TWINS_MQTT_URL: ${MF_TWINS_MQTT_URL}
+    ports:
+      - ${MF_TWINS_HTTP_PORT}:${MF_TWINS_HTTP_PORT}
+    networks:
+      - mainflux-base-net
+    env_file:
```

Comment:
This is necessary?

Reply:
I would also avoid additional .env file. If you must put it in global .env twins section

Reply:
It's not an additional one, it's the same. My question is if it really works if you run the composition from a different location than where the .env is.

Reply:
@manuio Its called `.env-twins` its a new file. He used both .env and new .env-twins no ?

Reply:
Right, I didn't see that he use both. In any case no need to specify the `.env` path